meetings/2024/README.md

```diff
@@ -39,18 +29,35 @@ All schedule items must have a public issue or checked-in proposal that can be l
 
 ### Mon Apr 1, 2024
 
+- [Implementation specific documentation](https://github.com/dotnet/csharplang/issues/7898). (Bill, on a Monday)
+- _Open slot_
+
 ### Wed Mar 27, 2024
 
+- Discriminated Union WG (Matt)
+
 ### Mon Mar 11, 2024
 
+- Meeting on meeting (Mads)
+
 ### Mon Mar 4, 2024
 
+- [Non-enumerable collection types](https://github.com/dotnet/csharplang/pull/7895) (Julien)
+- [Overload resolution priority](https://github.com/dotnet/csharplang/pull/7906) (Fred)
+
 ### Wed Feb 28, 2024
 
+- Extensions (Julien and Mads)
+
 ### Mon Feb 26, 2024
 
+- Review [metadata](https://github.com/dotnet/csharplang/blob/main/proposals/params-collections.md#metadata) section for"Params Collections". (Aleksey)
+- Review: [Allow lambdas with parameter modifiers, but without parameter types](https://github.com/dotnet/csharplang/pull/7369) (Cyrus)
+
 ### Wed Feb 21, 2024
 
+- Dictionary-expressions + Collection-expressions future: WG to go over our plans, and the initial design space (Cyrus)
```

Comment:
I'd like to attend for this topic if possible, but I cannot make it 2/21.

Reply:
Could we consider switching the agenda items for 2/21 and 2/26, if that works for @CyrusNajmabadi and @AlekseyTs?

Reply:
Works for me.

Reply:
@AlekseyTs will Feb 26 work for you?